.github/workflows/swift_matrix.yml

```diff
@@ -159,7 +159,6 @@ jobs:
           - image: swift:6.0-windowsservercore-ltsc2022
             swift_version: "6.0"
             enabled: ${{ inputs.matrix_windows_6_0_enabled }}
-    if: ${{ inputs.matrix_windows_6_0_enabled }}
```

Comment:
These if checks caused the naming problems since those are evaluated before the matrix so if we disabled the check then the names would get bad